scripts/show_parse.py

```diff
@@ -55,7 +56,20 @@ def print_parse(source: str, verbose: bool = False) -> None:
 
 def main() -> None:
     args = parser.parse_args()
-    print_parse(" ".join(args.program), args.verbose)
+    program = " ".join(args.program)
+    if args.grammar_file:
+        sys.path.insert(0, os.curdir)
+        from pegen.build import build_parser_and_generator
+
+        build_parser_and_generator(args.grammar_file, "pegen/parse.c")
```

Comment:
Due to passing "pegen/parse.c" as the filename, one can only execute the script from the root directory of the project. If not, then just a `FileNotFoundError` is thrown, which might be a bit misleading. Then again we may be the only ones using the script, so it's okay.

Reply:
The same is done by two other scripts; I'd rather not complicate the script without reason. If we find this is confusing users we can do something better (like computing the correct directory based on sys.argv[0]).